File: medplum-java/medplum-model/src/main/java/com/fhir/StructureDefinition_Context.java
package com.fhir;

import com.StagedBuilderStyle;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import java.lang.String;
import java.util.List;
import java.util.Optional;
import org.immutables.value.Value;

@Value.Immutable
@StagedBuilderStyle
@JsonDeserialize(
    as = ImmutableStructureDefinition_Context.class
)
@JsonIgnoreProperties(
    ignoreUnknown = true
)
public interface StructureDefinition_Context {
  Optional<List<Extension>> modifierExtension();

  Optional<List<Extension>> extension();

  Optional<String> expression();

  Optional<String> id();

  Optional<Structuredefinition_contextType> type();

  static ImmutableStructureDefinition_Context.Builder builder() {
    return ImmutableStructureDefinition_Context.builder();
  }
}

File: medplum-typescript/packages/sdk/src/model/fhir/StructureDefinition_Context.ts
import { Extension } from "./Extension";
import { Structuredefinition_contextType } from "./Structuredefinition_contextType";

export interface StructureDefinition_Context {
    id: string | null | undefined;
    extension: Extension[] | null | undefined;
    modifierExtension: Extension[] | null | undefined;
    type: Structuredefinition_contextType | null | undefined;
    expression: string | null | undefined;
}
